errors/daemon.go

```diff
@@ -827,4 +827,13 @@ var (
 		Description:    "While trying to delete a container, there was an error trying to delete one of its volumes",
 		HTTPStatusCode: http.StatusInternalServerError,
 	})
+
+	// ErrorVolNameConflict is generated when an error occurred while
+	// trying to create a volume that has existed using different driver.
+	ErrorVolNameConflict = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
I think this should be `ErrorVolumeNameTaken` better, with the value `VOLUME_NAME_TAKEN`.